ListItemFormatter.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'ListItemFormatter'
-  s.version = '0.0.1'
+  s.version = '0.1.0'
```

Comment:
No need to bump any version numbers, this is all handled by a fastlane lane that I can run on `master` once ready

Reply:
Thank you for letting me know, I am sorry to throw bunch of irrelevant changes 😂

Reply:
No worries, I should probably write some contributing guides 😀